cmd/commands/cli/command.go

```diff
@@ -356,6 +357,47 @@ func (c *cliApp) unlock(argsString string) {
 	success(fmt.Sprintf("Identity %s unlocked.", identity))
 }
 
+func (c *cliApp) payout(argsString string) {
+	args := strings.Fields(argsString)
+
+	const usage = "payout command:\n    set"
+	if len(args) < 1 {
+		info(usage)
+		return
+	}
+
+	action := args[0]
+	switch action {
+	case "set":
+		payoutSignature := "payout set <identity> <ethAddress>"
+		if len(args) < 2 {
+			info("Press tab to select identity.\n", payoutSignature)
```

Comment:
The information should be expanded upon here, you don't really need to press tab, you can enter it manually as well... So maybe something like: "Please provide an identity. You can select one by pressing tab.". It's more generic and useful in both cases.

Reply:
expanded message